plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -156,9 +156,6 @@ func NodeRules() []rbacv1.PolicyRule {
 		// Leases
 		rbacv1helpers.NewRule("get", "create", "update", "patch", "delete").Groups("coordination.k8s.io").Resources("leases").RuleOrDie(),
 
-		// CSI
-		rbacv1helpers.NewRule("get").Groups(storageGroup).Resources("volumeattachments").RuleOrDie(),
```

Comment:
A similar PR is https://github.com/kubernetes/kubernetes/pull/124472.

waiting for @liggitt review.

cc @jsafrane

Reply:
> we no longer bind the system:node role by default, but upgraded clusters that never enabled the node authorizer (because they don't care about node isolation or because it increases control plane memory/cpu use or because they just upgraded without reading release notes) still rely on the node role to grant kubelets permissions

should probably be get/list/watch?

Reply:
@liggitt we only need the get and watch permissions on a volumeattachment object.

Reply:
ah, ok ... so the kubelet is doing single-item get + watch. This lgtm then.